scrapyd/default_scrapyd.conf

```diff
@@ -10,6 +10,8 @@ finished_to_keep = 100
 poll_interval = 5.0
 bind_address = 127.0.0.1
 http_port   = 6800
+username    =
+password    =
```

Comment:
I think we can use a single config option
`http_auth = user:pass`
No worries for colon character in username
according to [rfc 7617](https://tools.ietf.org/html/rfc7617#section-2)
> Furthermore, a user-id containing a colon character is invalid, [...]
User-ids containing colons cannot be encoded in user-pass strings

Reply:
1. The `http_auth` option might be confusing as the configuration file of Scrapyd is not self-documented.
2. We should check the `username` option to avoid potential mistake (see [commit 8664360](https://github.com/scrapy/scrapyd/pull/326/commits/866436005f09a2df1b99b0a53e4e67a5e46bf0a6)), just like [htpasswd](https://httpd.apache.org/docs/2.4/programs/htpasswd.html):
```
$ htpasswd .htpassfile user:name
htpasswd: username contains illegal character ':'
```